src/main/java/com/mitchej123/hodgepodge/mixins/late/thaumcraft/MixinBlockMagicalLog.java

```diff
@@ -0,0 +1,40 @@
+package com.mitchej123.hodgepodge.mixins.late.thaumcraft;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockRotatedPillar;
+import net.minecraft.block.material.Material;
+import net.minecraft.world.World;
+
+import org.spongepowered.asm.mixin.Mixin;
+
+import thaumcraft.common.blocks.BlockMagicalLog;
+
+@Mixin(value = BlockMagicalLog.class)
+public abstract class MixinBlockMagicalLog extends BlockRotatedPillar {
+
+    protected MixinBlockMagicalLog(Material p_i45425_1_) {
+        super(p_i45425_1_);
+    }
+
+    /**
+     * Old breakBlock() does not override the correct method.
+     */
+    @Override
```

Comment:
Did you mean `@Overwrite` here?

Reply:
No. `BlockMagicalLog` does not actually override `breakBlock()` and I override it here